addons/mail/tests/test_link_preview.py

```diff
@@ -21,9 +22,40 @@ def _patched_get_html(*args, **kwargs):
     </head>
     </html>
     """
+    # To handle chunks read on stream requests
+    response.raw = io.BytesIO(response._content)
     response.headers["Content-Type"] = 'text/html'
     return response
 
+def _patched_get_html_without_og_title(*args, **kwargs):
```

Comment:
Any reason (except first patched done like that) defining those global and not in the test class itself ? I prefer to keep things local when possible :)